src/tools/interop/idt/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+FROM debian:bookworm
+
+# deps
+RUN apt-get update
+RUN apt-get install -y apt-utils git wget
+# python
+RUN apt-get install -y python3.11 python3-pip python3.11-venv
+# pybluez
+RUN apt-get install -y pkg-config glib-2.0 libboost-python-dev libboost-thread-dev libbluetooth-dev libglib2.0-dev
+
+# adb
+RUN apt-get install -y usbutils adb
```

Comment:
Here is are some steps that I've used to install the latest platform tools when not available in apt.

`1949` is the vendor ID for Amazon devices

```bash
sudo su

cat > /etc/apt/sources.list.d/amd64.list <<EOF
deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse
deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted universe multiverse
deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-backports main restricted universe multiverse
EOF

apt-get update
apt-get upgrade
apt-get install openjdk-19-jdk binfmt-support qemu-user-static
update-binfmts --display
dpkg --add-architecture amd64
apt install libc6:amd64

curl -o commandlinetools.zip https://dl.google.com/android/repository/commandlinetools-linux-9477386_latest.zip
unzip commandlinetools.zip

export ANDROID_HOME=/opt/android-sdk
yes | cmdline-tools/bin/sdkmanager --sdk_root=${ANDROID_HOME} --licenses
cmdline-tools/bin/sdkmanager --sdk_root=${ANDROID_HOME} --install "cmdline-tools;latest" "platform-tools" "ndk;23.2.8568313"

cat >> /etc/udev/rules.d/51-android.rules <<EOF
SUBSYSTEM=="usb", ATTR{idVendor}=="1949", MODE="0666", GROUP="plugdev"
EOF

reboot
```